content/html/concepts/elements/terms/del/del.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '<del>'
+Description: 'Used to show deleted portion text by a document, usually rendered in strikethrough text.'
+Subjects:
+- 'Web Development'
+- 'Web Design'
+  Tags:
+- 'Elements'
+- 'Tags'
+- 'Style'
+  CatalogContent:
+- 'learn-html'
+- 'paths/front-end-engineer-career-path'
+---
+
+The **`<del>`** element is used to show a deleted portion of text and is commonly intended to show the changes made to a document while keeping a history of the changes made. Browsers render it showing strike-through text. Its behavior equals the `<s>` element but is semantically different, `<s>` is usually used to show no longer pertinent, inaccurate, or wrong text.
+
+## Syntax
+
+```html
+<del>
+  <!-- deleted text here -->
+</del>
+```
+
+The **`<del>`** element should only contain text content or other elements meant for text content.
+
+## Example
+
+```html
+<html>
+  <head> </head>
+  <body>
+    <p>It show <del>deleted</del> text</p>
```

Comment:
```suggestion
    <p>This is <del>deleted</del> text.</p>
```

Reply:
I would make this edit and replace with a new screen `.png` image.

Reply:
done

Reply:
Done